config/prod/gene_drug_db_test_instance.yaml

```diff
@@ -0,0 +1,20 @@
+# Provision an EC2 instance
+template_path: templates/managed-ec2-v2.yaml
+stack_name: gene_drug_db_test_instance
+parameters:
+  # The Sage deparment for this resource
+  Department: "Platform"
```

Comment:
is this your Department and Project?

Reply:
That makes sense :) 👍 It should be fixed with my new commit.